src/api/controller/api/upload.js

```diff
@@ -72,14 +72,12 @@ export const uploadFile = ctx => async loaderName => {
     const parsedStream = parseStream(mergedStream);
     try {
         await ctx.saveParsedStream(parsedStream, ctx.initDatasetUri);
+        progress.finish();
```

Comment:
Why didn't you finish "progress" after "clearChunks" like before ?

Reply:
The goal is to clearChunks even if progress hadn't finished correctly
Because previously, went something when wrong, chunks were staying, en were taken twice at the next upload

Reply:
ok :+1: